src/plugins/chatwizard/_base.scss

```diff
@@ -217,6 +221,10 @@
 	width: 40px;
 }
 
+.wb-chtwzrd-container .reset {
+	right: 42px;
+}
+
```

Comment:
You will also have to add this rule for when the chat wizard is inline (contained within the container), since the minimize button isn't visible, this one takes its spot:
```
.wb-chtwzrd-contained .reset {
	right: 0;
}
```